tests/ModelTest.php

```diff
@@ -572,4 +572,15 @@ public function testChunkById(): void
 
         $this->assertEquals(3, $count);
     }
+
+    public function toSqlTest(): void
+    {
+        $this->assertTrue(
+            is_string(User::where('age', 30)->toSql())
+        );
+
+        $this->assertTrue(
+            is_string(User::whereRaw(['age' => ['$gt' => 30, '$lt' => 40]])->toSql())
```

Comment:
Add additional test case with assert that result equal to generated string (for example `select * from user where age ...`)

Reply:
Fixed in https://github.com/jenssegers/laravel-mongodb/pull/1916/commits/1ad945e303fb9a527db5eb1d7aa4a0606473c3f2